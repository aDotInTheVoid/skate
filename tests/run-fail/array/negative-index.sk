fn main () {
    let a = 1;
    print [1,2,3][-a];
}
