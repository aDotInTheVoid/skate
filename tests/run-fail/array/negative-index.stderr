error: Expected a positive number
  ┌─ $DD/tests/run-fail/array/negative-index.sk:3:19
  │
3 │     print [1,2,3][-a];
  │                   ^^ Evaluated to `-1`

